qemu/tests/cfg/check_block_size.cfg

```diff
@@ -88,3 +88,17 @@
             nic_mode = tap
             physical_block_size_stg = 512
             logical_block_size_stg = 512
+        - 4096_4096_reboot:
```

Comment:
suggest to move under - 4096_4096
like 
- 4096_4096
     ovmf:
        - normal_cluster_size
        - 4069_cluster_size

Reply:
4096 installation not support on bios
python ConfigTest.py --testcase=check_block_size --guestname=RHEL.9.0.0  --firmware=default_bios

(1/5) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: STARTED
 (1/5) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (672.85 s)
 (2/5) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.check_block_size.4096_4096.base.q35: STARTED
 (2/5) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.check_block_size.4096_4096.base.q35: PASS (35.38 s)
 (3/5) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.check_block_size.4096_512.extra_cdrom_ks.q35: STARTED
 (3/5) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.check_block_size.4096_512.extra_cdrom_ks.q35: PASS (825.90 s)
 (4/5) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.check_block_size.512_512.extra_cdrom_ks.q35: STARTED
 (4/5) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.check_block_size.512_512.extra_cdrom_ks.q35: PASS (830.60 s)
 (5/5) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.check_block_size.4096_4096_reboot.extra_cdrom_ks.q35: STARTED
 (5/5) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.check_block_size.4096_4096_reboot.extra_cdrom_ks.q35: ERROR: Login timeout expired    (output: 'exceeded 360.0 s timeout, last failure: Could not verify DHCP lease: 9a:66:f7:7d:73:42 --> 10.73.198.208') (1212.74 s)
RESULTS    : PASS 4 | ERROR 1 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/kar/workspace/job-results/job-2022-08-01T08.21-29a6d98/results.html
JOB TIME   : 3582.26 s